Source/igtlMessageFactory.cxx

```diff
@@ -75,6 +76,7 @@ MessageFactory::MessageFactory()
   this->AddMessageType("RTS_QTDATA", (PointerToMessageBaseNew)&igtl::RTSQuaternionTrackingDataMessage::New);
   this->AddMessageType("STT_QTDATA", (PointerToMessageBaseNew)&igtl::StartQuaternionTrackingDataMessage::New);
   this->AddMessageType("STP_QTDATA", (PointerToMessageBaseNew)&igtl::StopQuaternionTrackingDataMessage::New);
+  this->AddMessageType("GET_IMGMETA", (PointerToMessageBaseNew)&igtl::ImageMetaMessage::New);
```

Comment:
For other message types, X and GET_X are two different messages, for example:

    this->AddMessageType("POINT", (PointerToMessageBaseNew)&igtl::PointMessage::New);
    this->AddMessageType("GET_POINT", (PointerToMessageBaseNew)&igtl::GetPointMessage::New);

Is there a particular reason why we don't have something like this?

    this->AddMessageType("IMGMETA", (PointerToMessageBaseNew)&igtl::ImageMetaMessage::New); 
    this->AddMessageType("GET_IMGMETA", (PointerToMessageBaseNew)&igtl::GetImageMetaMessage::New);

Reply:
I realized my mistake when I was testing the code. I made the changes locally, but I haven't updated the branch yet.